src/flood_model/bg_flood_model.py

```diff
@@ -398,16 +398,13 @@ def run_bg_flood_model(
     # Get the valid BG-Flood Model directory
     bg_flood_dir = get_valid_bg_flood_dir()
     # Get the file path of the Hydro DEM for the catchment area
-    hydro_dem_path = dem_metadata_in_db.get_catchment_hydro_dem_filepath(engine, catchment_area)
-    # Use the resolution of the Hydro DEM if it is not provided
-    if resolution is None:
-        _, resolution = dem_metadata_in_db.get_hydro_dem_data_and_resolution(engine, catchment_area)
```

Comment:
As discussed in meeting, this resolution should be gathered from either the hydrological DEM or if you think it is a waste of resources then it should be read from a new column in the database. It shouldn't be deleted.

Reply:
done